docs/install/airgap.md

```diff
@@ -40,3 +44,34 @@ system-default-registry: "registry.example.com:5000"
 ```
 
 **Note:** The `system-default-registry` parameter must specify only valid RFC 3986 URI authorities, i.e. a host and optional port.
+
+### RKE2 Install.sh Script Install
+
+You may use the `install.sh script` and define the `INSTALL_RKE2_ARTIFACT_PATH` to reduce steps and create the systemd service file.
+
+1. Download the rke2, rke2-images, and sha256sum archives from the release into a directory, as in the example below:
+
+```
+# mkdir /root/rke2-artifacts && cd /root/rke2-artifacts/
+# curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2-images.linux-amd64.tar.zst
+# curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2.linux-amd64.tar.gz
+# curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/sha256sum-amd64.txt
+# curl -sfL https://get.rke2.io --output install.sh
+```
```

Comment:
Just include the commands to run, and without a # prefix so that users can just copy-paste the commands.
```suggestion
```bash
mkdir /root/rke2-artifacts && cd /root/rke2-artifacts/
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2-images.linux-amd64.tar.zst
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/rke2.linux-amd64.tar.gz
curl -OLs https://github.com/rancher/rke2/releases/download/v1.21.5%2Brke2r2/sha256sum-amd64.txt
curl -sfL https://get.rke2.io --output install.sh
```
```